gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1292,6 +1322,22 @@ MM_ParallelGlobalGC::fixHeapForWalk(MM_EnvironmentBase *env, UDATA walkFlags, ui
 	return fixedObjectCount;
 }
 
+void
+MM_ParallelGlobalGC::clearHeap(MM_EnvironmentBase *env, MM_HeapWalkerObjectFunc walkFunction)
+{
+	HeapSizes counter;
+	counter.freeBytes = 0;
+	counter.objectBytes = 0;
+	OMRPORT_ACCESS_FROM_ENVIRONMENT(env);
+	_heapWalker->allObjectsDo(env, walkFunction, &counter, MEMORY_TYPE_RAM, false, false, true);
+	U_64 startTime = omrtime_hires_clock();
+	_extensions->globalGCStats.fixHeapForWalkTime += omrtime_hires_delta(startTime, omrtime_hires_clock(), OMRPORT_TIME_DELTA_IN_MICROSECONDS);
+	Assert_MM_true(FIXUP_NONE != _extensions->globalGCStats.fixHeapForWalkReason);
+	_extensions->globalGCStats.fixHeapForWalkReason = FIXUP_AND_CLEAR_HEAP;
+	Trc_MM_clearheap(env->getLanguageVMThread(), counter.freeBytes, counter.objectBytes);
+	Assert_MM_true(counter.freeBytes + counter.objectBytes == _extensions->heap->getMemorySize());
```

Comment:
this code block is a bit too dense - add a couple of empty lines to split 3 sections: prolog with declarations, body that does something useful, epilog that report results/trace point)